rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -436,3 +436,36 @@ def _list_oflow_count_for_sandboxes(self, sandboxes,
             "chart_plugin": "StackedArea", "data": oflow_data
         }
         self.add_output(additive_oflow_data)
+
+    def _create_address_set(self, set_name, address_list):
+        LOG.info("create %s address_set [%s]" % (set_name, address_list))
+
+        name = "name=\"" + set_name + "\""
+        addr_list="\"" + address_list + "\""
+
+        ovn_nbctl = self.controller_client("ovn-nbctl")
+        ovn_nbctl.set_sandbox("controller-sandbox", self.install_method)
+        ovn_nbctl.enable_batch_mode()
```

Comment:
If only one command is going to be executed, then batch mode is not needed (batch mode is off by default), and the flush() is not needed either.